sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -100,10 +100,10 @@ static inline std::string codeToString(cl_int code) {
 #endif // #ifdef __SYCL_DEVICE_ONLY__
 
 #define __SYCL_OCL_ERROR_REPORT                                                \
```

Comment:
I think `__SYCL_OCL_ERROR_REPORT` macro should be updated as well.

`__SYCL_OCL_ERROR_REPORT` -> `__SYCL_NATIVE_ERROR_REPORT`
`__SYCL_SUPPRESS_OCL_ERROR_REPORT` -> `__SYCL_SUPPRESS_NATIVE_ERROR_REPORT`

Probably it make sense to keep support for `__SYCL_SUPPRESS_OCL_ERROR_REPORT` in addition for a while, but report a message that it's deprecated and is going to be removed.

Update: it applies to all defines in `common.hpp` file with `_OCL` in the name: `_OCL` -> `_NATIVE`.

Reply:
Sure. There are so many internals that need to be renamed from OpenCL, and I want to just quickly fix what is visible to the user. Let's not have this gated by other renaming, which I currently don't have bandwidth to work on.

Reply:
Okay. Let's leave a TODO comment or GitHub issue to address this later.

Reply:
Added TODO comment.